add support for overarc

srkondam opened the pull request:
add support for overarc..

srkondam commented:
@eluberoff 

I have added a test page air-demo.html to all my work.

eluberoff commented on src/css/math.less:
```diff
@@ -343,6 +343,13 @@
+    border-top: 1px solid black;
+    border-radius: 50% 50% 0 0;
+    padding: 0.35em 0.25em 0 0.1em !important;
+    margin-top: 0.1em !important;
```
do these need the `!important` modifiers? we generally try to avoid those whenever possible

srkondam replied:
modifier is not required.

srkondam commented:
added a changeset with suggested changes..

jwmerrill commented:
What LaTeX package is `\overarc` defined in?

We've generally tried to stick with base LaTeX commands, or things that are in amsmath or amssymb. I don't think `\overarc` is in any of these, and I couldn't quickly find where it comes from.

I think sticking to a widely used subset of LaTeX is helpful, because it means that the LaTeX that MathQuill emits will be more likely to be interpreted successfully by other systems.

Sorry to be late to the party on this one.

jwmerrill commented:
I see that it comes from the [arcs](http://mirrors.sorengard.com/ctan/macros/latex/contrib/arcs/arcs.pdf) package.

jwmerrill commented:
Could you say more about the context where you're going to use this? What concept is it used to represent?

jwmerrill commented on test/air-demo.html:
```diff
@@ -0,0 +1,58 @@
+    </script>
+</body>
+
+</html>
```
What is the purpose of this demo page? I'm not 100% sure it belongs in this repo.

srkondam commented:
Jason,

We are trying to add "arc" operators from the attached image document.

On Mon, Feb 26, 2018 at 8:28 PM, Jason Merrill <notifications@github.com>
wrote:

> Could you say more about the context where you're going to use this? What
> concept is it used to represent?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/desmosinc/mathquill/pull/94#issuecomment-368725043>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AUAYKgGXWl6xTD2_2T4nHwyI5N7lUxQ0ks5tY2g_gaJpZM4SRW2w>
> .
>



-- 
Thanks,
Sandeep.

eluberoff commented:
Can you attach the image? It's not included. You may need to add it to
github directly. You can also directly send it to jason@desmos.com and
eli@desmos.com if that's easier.

I agree with Jason that we shouldn't merge in any non-standard LaTeX
commands.

On Mon, Feb 26, 2018 at 6:50 PM, srkondam <notifications@github.com> wrote:

> Jason,
>
> We are trying to add "arc" operators from the attached image document.
>
> On Mon, Feb 26, 2018 at 8:28 PM, Jason Merrill <notifications@github.com>
> wrote:
>
> > Could you say more about the context where you're going to use this? What
> > concept is it used to represent?
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/desmosinc/mathquill/pull/94#issuecomment-368725043>,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/AUAYKgGXWl6xTD2_
> 2T4nHwyI5N7lUxQ0ks5tY2g_gaJpZM4SRW2w>
> > .
> >
>
>
>
> --
> Thanks,
> Sandeep.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/desmosinc/mathquill/pull/94#issuecomment-368728937>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAdMe6HJk55C6NpMkabGOrPRDAri5ZfGks5tY21xgaJpZM4SRW2w>
> .
>

srkondam commented:
![operators](https://user-images.githubusercontent.com/20977706/36708186-7c056726-1b37-11e8-8e1e-a20a15678928.png)

srkondam commented:
Added image on github.

@Jason, I added air demo page for custom work. I will delete the page from
repo.

On Mon, Feb 26, 2018 at 8:53 PM, Eli Luberoff <notifications@github.com>
wrote:

> Can you attach the image? It's not included. You may need to add it to
> github directly. You can also directly send it to jason@desmos.com and
> eli@desmos.com if that's easier.
>
> I agree with Jason that we shouldn't merge in any non-standard LaTeX
> commands.
>
> On Mon, Feb 26, 2018 at 6:50 PM, srkondam <notifications@github.com>
> wrote:
>
> > Jason,
> >
> > We are trying to add "arc" operators from the attached image document.
> >
> > On Mon, Feb 26, 2018 at 8:28 PM, Jason Merrill <notifications@github.com
> >
> > wrote:
> >
> > > Could you say more about the context where you're going to use this?
> What
> > > concept is it used to represent?
> > >
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/desmosinc/mathquill/pull/94#issuecomment-368725043
> >,
> > > or mute the thread
> > > <https://github.com/notifications/unsubscribe-auth/AUAYKgGXWl6xTD2_
> > 2T4nHwyI5N7lUxQ0ks5tY2g_gaJpZM4SRW2w>
> > > .
> > >
> >
> >
> >
> > --
> > Thanks,
> > Sandeep.
> >
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/desmosinc/mathquill/pull/94#issuecomment-368728937>,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/
> AAdMe6HJk55C6NpMkabGOrPRDAri5ZfGks5tY21xgaJpZM4SRW2w>
> > .
>
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/desmosinc/mathquill/pull/94#issuecomment-368729429>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AUAYKucDBpufhUj3ZaUehjrbhcImjoP1ks5tY24TgaJpZM4SRW2w>
> .
>



-- 
Thanks,
Sandeep.

jwmerrill commented:
Ok, thanks a lot for following up.

I haven't been able to find any more commonly used way of adding this symbol (although it sounds like `\overparen` from yhmath is maybe equally as common), so I think we should go with `\overarc`.
